cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/documentstore/ConflictResolver.java

```diff
@@ -17,31 +17,35 @@
 import java.util.List;
 
 /**
- * Interface to implement conflicts resolving algorithm.
+ * The {@code ConflictResolver} interface should be implemented by classes which are used to resolve
```

Comment:
What about `...interface should be implemented by classes which resolve conflicts via...`